README.md

```diff
@@ -320,9 +320,11 @@ Install Galaxy on your local system with all the default options:
     galaxy_server_dir: /srv/galaxy
   connection: local
   roles:
-     - galaxy
-```
-
+     - galaxyproject.galaxy
+```  
+  
+If your Ansible version >= 2.10.4, then you must indicate explicitly ansible_user in your hosts file, otherwise you will gen an error  
+  
```

Comment:
```suggestion
If your Ansible version >= 2.10.4, then when you run `ansible-playbook playbook.yml` you should supply an extra argument `-u $USER`, otherwise you will get an error.
  
```